akka-docs/rst/project/migration-guide-2.3.x-2.4.x.rst

```diff
@@ -324,6 +324,30 @@ The ``permanent`` flag in ``deleteMessages`` was removed. non-permanent deletes
 any more. Events that were deleted with ``permanent=false`` with older version will
 still not be replayed in this version.
 
+Recover() a method, not message; can not be sent from external source
```

Comment:
make the heading shorter, and explain things in the text, i.e. move "can not be sent from external source"